bin/setup

```diff
@@ -2,26 +2,26 @@
 
 # Run this script immediately after cloning the codebase.
 
-#!/usr/bin/env sh
-
-# Run this script immediately after cloning the codebase.
+# Squash all commits to get a clean slate
+git update-ref -d refs/heads/master
+git commit -am "New Proteus-Gulp project"
 
-# Make sure Bundler is installed
+# Make sure npm is installed
 if ! type npm > /dev/null; then
- echo -e "Installing NPM..."
+ echo "Installing npm..."
  curl -L https://npmjs.org/install.sh | sh
 fi
 
-# Set up NPM dependencies
-echo -e "Installing Dependencies..."
+# Set up npm dependencies
+echo "Installing Dependencies..."
 npm install
 
 # Remove Git remote if it's still the proteus-middleman repo
-if [ `git config --get remote.origin.url` = "git@github.com:thoughtbot/proteus-gulp.git" ]; then
-  echo -e "What is your repo url? Enter URL or leave blank"
+if [ "$(git config --get remote.origin.url)" = "git@github.com:thoughtbot/proteus-gulp.git" ]; then
+  echo "What is your repo url? Enter URL or leave blank"
   read url
-  if [ ! $url = '' ]; then
+  if [ ! "$url" = "" ]; then
```

Comment:
There is actually a built-in way to check for empty strings!

```bash
if [ -z "$url" ]; then
```

More here: http://tldp.org/LDP/abs/html/comparison-ops.html